build_for_ultimaker.sh

```diff
@@ -6,8 +6,27 @@
 # Copyright (C) 2018 Olliver Schinagl <oliver@schinagl.nl>
 #
 
+CI_REGISTRY_IMAGE="${CI_REGISTRY_IMAGE:-registry.gitlab.com/olliver/um-kernel}"
+CI_REGISTRY_IMAGE_TAG="${CI_REGISTRY_IMAGE_TAG:-latest}"
+
+WORKDIR="${WORKDIR:-/build}"
+
 set -eu
 
 git submodule update --init --recursive
 
-./build.sh "${@}"
+if ! command -V docker; then
+	echo "Docker not found, attempting native build."
+
+	./build.sh "${@}"
```

Comment:
I would prefer removing the try to build native, if you are an expert user you can call the build.sh file yourself. 
Just echo them docker is not installed and what should be installed.

Reply:
I think we need a new ticket/issue that replaces build with exit altogether (and replaces the text) to 'force' our users